modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -143,9 +156,15 @@ def find_lighthouse_claim!(claim_id:)
         def find_bgs_claim!(claim_id:)
           return if claim_id.blank?
 
-          bgs_service.ebenefits_benefit_claims_status.find_benefit_claim_details_by_benefit_claim_id(
-            benefit_claim_id: claim_id
-          )
+          if Flipper.enabled? :bgs_via_faraday
```

Comment:
Do we still need to check the Flipper feature at this point, or are we committed to using the Faraday approach?

Reply:
yeah, we are safe to remove the Flipper check. Our initial logic with Flipper was set to introduce this in a safe manner. now that we know the connection is working, lets remove this going forward.